src/nav-creator.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable import/no-cycle */
+import createMainPage from './main-page';
+import createContactPage from './contact-creator';
+import createMenuPage from './menu-creator';
+/* eslint-enable import/no-cycle */
+
+function createNavigation() {
+  const content = document.getElementById('content');
+
+  const linkMain = '<a href="#" class="main">Go to Main Page</a>';
+  const linkMenu = '<a href="#" class="menu">Go to Menu Page</a>';
+  const linkContact = '<a href="#" class="contact">Go to Contacts Page</a>';
+
+  content.innerHTML += linkMain;
+  content.innerHTML += linkMenu;
+  content.innerHTML += linkContact;
+}
+
+function addAllEventListeners() {
```

Comment:
- [ ] [required] You should use ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))
  - [ ] [required] Usage of arrow functions instead of the `function` keyword